includes/AppLeave.php

```diff
@@ -141,8 +144,8 @@
 	//echo "Updating application for $user.  ";
 	
 	$userDB = str_replace('.','$',$user);
-	$stmt = $sqlConn->prepare("INSERT INTO $userDB VALUES (?,?)");
-	$stmt->bind_param('ss',$id,$type);
+	$stmt = $sqlConn->prepare("INSERT INTO $userDB VALUES (?,?,?)");
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->